gems/activerecord/6.0/activerecord.rbs

```diff
@@ -431,8 +431,8 @@ module ActiveRecord
                    | (?batch_size: Integer, ?start: Integer, ?finish: Integer, ?error_on_ignore: bool) -> Enumerator[Model, nil]
       def find_in_batches: (?batch_size: Integer, ?start: Integer, ?finish: Integer, ?error_on_ignore: bool) { (Array[Model]) -> void } -> nil
       def in_batches: (?of: Integer, ?start: Integer, ?finish: Integer, ?load: bool, ?error_on_ignore: bool, ?order: untyped) { (self) -> void } -> nil
-      def sum: (?untyped? column_name) -> Integer
-             | () { (Model) -> Integer } -> Integer
+      def sum: (?untyped? column_name) -> untyped
+             | [T] () { (Model) -> T } -> (Integer | T)
```

Comment:
The intent of my comment was to use overloading.
I don't know if it will work.

```suggestion
      def sum: (?untyped? column_name) -> untyped
             | ...
```

Reply:
I didn’t realize that overloading could be used here, but that makes perfect sense. I'll check it out.

Reply:
It seems to result in an `InvalidOverloadMethodError`. 😢